src/GNDStk/mixed/src/detail.hpp

```diff
@@ -0,0 +1,160 @@
+
+namespace detail {
+
+// text_metadatum_to_string
+// Helper for cdata and comment
+class text_metadatum_to_string {
```

Comment:
Where is this stuff tested?

Reply:
In the current invocation of the test suite, it looks like that particular detail material, `text_metadatum_to_string`, indeed wasn't being tested, even indirectly. So, I put in a test (indirect, but the function does get called now).

Other functions in that file do, in fact, get tested indirectly. I have it in mind to actually change some of that functionality works, and will put that in another PR sometime. Because of that, it probably isn't worthwhile at this time to write direct tests that'll just get tossed after I change the constructs in question.